pyproject.toml

```diff
@@ -66,6 +66,8 @@ lint.ignore = [
     "EM102",  # https://docs.astral.sh/ruff/rules/f-string-in-exception/
     "TRY003",  # https://docs.astral.sh/ruff/rules/raise-vanilla-args/
     "D407",  # https://docs.astral.sh/ruff/rules/dashed-underline-after-section/
+    "COM812", # https://docs.astral.sh/ruff/rules/missing-trailing-comma/
```

Comment:
Just a vote for keeping this rule iso ingoring - but leave it to you

Reply:
I also like this rule, but the `ruff` formatter recommended dropping it for compatability, so I am just obeying. :robot: